spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/AckMode.java

```diff
@@ -24,14 +24,17 @@
 public enum AckMode {
 
 	/**
-	 * The framework does not ack/nack and the
-	 * {@link com.google.cloud.pubsub.v1.AckReplyConsumer} is sent back to the user.
+	 * The framework does not ack / nack the message. The
+	 * {@link com.google.cloud.pubsub.v1.AckReplyConsumer} is sent back to the user
+	 * for acking or nacking the {@link com.google.pubsub.v1.PubsubMessage}.
```

Comment:
That's does bring to much value. We have a link to the `AckReplyConsumer`, so, essentially you are repeating its JavaDocs here and that's already an end-user responsibility what to do with such a `AckReplyConsumer`.

But yeah... Let it. It's not so critical to spend more time on this.